Doc/Tutorial/chapter_pdb.tex

```diff
@@ -89,6 +89,32 @@ \subsection{Reading an mmCIF file}
 >>> y_list = mmcif_dict['_atom_site.Cartn_y']
 \end{verbatim}
 
+
+\subsection{Reading files in the MMTF format}
+
+You can use the direct MMTFParser to read a structure from a file:
+\begin{verbatim}
+>>> from Bio.PDB.mmtf import MMTFParser
+>>> structure = MMTFParser.get_structure("PDB/4CUP.mmtf")
+\end{verbatim}
+
+Or you can use the same class to get a structure by it's PDB ID:
+\begin{verbatim}
+>>> structure = MMTFParser.get_structure_from_url("4CUP")
+\end{verbatim}
+
+This gives you a Structure object as if read from a PDB or mmCIF file.
+
+You can also have access to the underlying data using the MMTF module.
```

Comment:
Potentially confusing to the reader, perhaps change to something like this?

...using the external MMTF library which Biopython is using internally:

Note this test would require internet access, and we currently don't have a mechanism in ``test_Tutorial.py` to handle that, so we can't use it as a doctest (yet).